resources/views/pages/home.blade.php

```diff
@@ -76,7 +77,7 @@ function gtag(){dataLayer.push(arguments);}
     <div class="container">
          <div class="expenses">
              <govt-expense></govt-expense>
-             <a href="{{route('expense.reports')}}" class="mt-4 mb-5 mx-auto">View Expenditure Report</a>
+             <a href="{{route('expense.reports')}}" class="mt-4 mb-5 mx-auto" target="_blank">View Expenditure Report</a>
```

Comment:
This is not an external link. I meant links that are redirecting to another website. Like Twitter